src/lint/rules/default/index.ts

```diff
@@ -0,0 +1,14 @@
+import { RulesList } from "types/rules";
+import subjectMaxLength from "./subject-max-length";
+import subjectMinLength from "./subject-min-length";
+import subjectCapitalizeFirst from "./subject-capitalize-first";
+import subjectNoEndPeriod from "./subject-no-end-period";
+
+const defaultRules: RulesList = [
+  subjectMaxLength,
+  subjectMinLength,
+  subjectCapitalizeFirst,
+  subjectNoEndPeriod,
+];
+
+export default defaultRules;
```

Comment:
What would be the purpose of 'default' rules?

As I understand `default` is an attribute of  a set of rules, so rules themselves do not have the distinction of whether it is default. Instead, some rules may be included in the default preset.

Therefore, I'll suggest moving the rules to one upper level, residing in `lint/rules`, and the rule presets are moved to `lint/preset.ts`, exporting a named constant of `defaultPreset`.

Reply:
good idea